docs/modules/ROOT/pages/tutorial-onpremise.adoc

```diff
@@ -1,145 +1,120 @@
 = On-premise example
 
-[abstract]
-CMOS is deployed to monitor an on-premise cluster with a simple Nginx web server exposed on port 8080 to provide a landing page.
-
 include::partial$tutorial.adoc[]
 
+== Overview
+
+Couchbase Monitoring and Observability Stack (also known as CMOS) is a simple, out-of-the-box solution built using industry standard tooling to observe the state of a Couchbase cluster.
+CMOS can be deployed in a VM using Docker or any container runtime.
+
 ifdef::env-github[]
 :imagesdir: https://github.com/couchbaselabs/observability/raw/main/docs/modules/ROOT/assets/images
 endif::[]
 
-.Microlith overview
+.Microlith runtime image
 image::microlith-runtime.png[]
 
-== Usage
+=== Installation
 
-To run refer to the xref:quickstart.adoc[quick start guide] for full details:
+At this moment, we only support deployment of CMOS using containers.
+Use docker to start the CMOS container.
+Alternatively, you can also use Linux container tools such as link:https://podman.io[Podman^].
+To install CMOS run the command below:
 
-[source, console]
+[source,console]
 ----
-docker run --name=cmos --rm -d -P couchbase-observability-stack
+docker run --name=cmos --rm -d -P couchbase/observability-stack:latest
 ----
 
-Use `docker ps` or `docker inspect X` to see the local ports exposed, the mapping to `3000` is the Grafana one so to get this:
+==== Port Configuration
 
-[source, console]
-----
-docker container port couchbase-grafana 3000
-0.0.0.0:55124
-:::55124
-----
+Couchbase Monitoring and Observability Stack contains multiple services, each running on their own default port such as Grafana(3000), Prometheus(9090), Alert manager(9093), Landing page(8080), Cluster monitor(77196), and Loki(3100).
```

Comment:
```suggestion
CMOS contains multiple services, each running on their own default port such as Grafana (3000), Prometheus (9090), Alertmanager (9093), Landing page (8080), and Loki (3100).
```

* Cluster Monitor's UI isn't publicly exposed for the DP
* `Alertmanager` is the official spelling
* I think it makes sense to spell out CMOS the first time it's used, then use the abbreviation afterward - @maria-robobug what do you think?

Reply:
Yep, using CMOS in subsequent references is fine 👍 .